### PR TITLE
Push to AWS, Azure, KVM, OpenStack, and VMWare

### PR DESCRIPTION
**Pull request by stone-z**
### Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -137,72 +137,71 @@ workflows:
...
               only: /^v.*/
 
-      # - architect/push-to-app-collection:
-      #     name: push-policies-shared-to-aws-app-collection
-      #     context: architect
-      #     app_name: "policies-shared"
-      #     app_namespace: "giantswarm"
-      #     app_collection_repo: "aws-app-collection"
-      #     requires:
-      #       - push-policies-shared-to-catalog
```
Comment (MarcelMue) [inline .circleci/config.yml:147]: Why not push to shared as well? Would be nicer later on.

Comment (stone-z) [reply .circleci/config.yml:147]: `policies-shared` was the name of the old app

Comment (stone-z) [reply .circleci/config.yml:147]: Ah do you mean https://github.com/giantswarm/shared-app-collection? Is that used? I'm fine to push there but wasn't sure what the delivery guarantees were at this point

Comment (MarcelMue) [reply .circleci/config.yml:147]: Delivery guarantees are not great AFAIK but there was agreement to use it.

Comment (stone-z) [reply .circleci/config.yml:147]: Added shared app collection as well

**Pull request merged**